tests/python/test_offline_cache.py

```diff
@@ -511,9 +511,7 @@ def run_simple_kernels(max_size):
         only_init(max_size)
         for kernel, args, get_res, num_offloads in simple_kernels_to_test:
             assert kernel(*args) == test_utils.approx(get_res(*args))
-            if curr_arch in [ti.vulkan]:
-                sleep(
-                    1)  # make sure the kernels are not used in the same second
+            sleep(1)  # make sure the kernels are not used in the same second
```

Comment:
@PGZXB sorry I  just noticed this `sleep` lol, would you mind explaining more detail why this sleep is necessary?

Reply:
Because both the `last_used_at` and `created_at` are timestamp in sec. To test the LRU/FIFO cache cleaning, we should make sure the kernels are used/created in different sec lol.

Reply:
This is because The cache cleaning policy chooses which kernel to be cleaned based on the last used time of the kernels. Different kernels have different number of files, and this affects the assertion below which checks how many files remains in the directory. How about let the last used time use millisecond instead of second? @PGZXB

Reply:
lgtm

Reply:
aha I see, then it sounds good to me if you just add a comment about the timestamp being at second precision in this PR and switch to millisecond in a followup PR. Up to you @PGZXB